create-entry.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
```

Comment:
Different systems put bash in different places. I prefer to use `/bin/sh`, which is portable, but if you decide to depend on bash features, at least use `#!/usr/bin/env bash`. For example, on my current OpenBSD laptop, I don't even have bash installed, but on most of the other systems in herd, it's in `/usr/local/bin/bash`

`/usr/bin/env` is required by the POSIX standard, and it will look up where bash is installed and run it.